.vscode/settings.json

```diff
@@ -1,4 +1,4 @@
 {
     "editor.formatOnSave": true,
     "rust-client.channel": "nightly-2022-10-22",
-}
+}
```

Comment:
Do we need this change? Should be newline terminated, no?

Reply:
This cange was modified by VS Code, however the text inside `settings.json` matches the exact content from tock's master, I do not know why it was marked as a modification, sorry for that.

Reply:
It was marked as a modification because vscode removed a newline from the end of the file

Reply:
It tock master there is no newline at the end of the file, however I will change and will add an ending newline to the settings.json